src/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.Tests.csproj

```diff
@@ -131,6 +131,7 @@
     <Compile Include="System\Runtime\InteropServices\VariantWrapperTests.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
+    <Compile Include="System\Runtime\CompilerServices\IDispatchConstantAttributeTests.cs"  Condition="'$(TargetsWindows)' == 'true'" />
```

Comment:
The preferred CoreFX style is to exclude the platform-specific tests with attributes, like `[PlatformSpecific(TestPlatforms.Windows)]`.

Reply:
Will do. Thanks.